taxonomies/categories.txt

```diff
@@ -21551,9 +21571,14 @@ nl:Chocoladecakes, Chocoladecake
 pt:Bolos de chocolate
 zh:巧克力蛋糕
 wikidata:en:Q30186
+ciqual_food_code:en:23585
+ciqual_food_name:en:Chocolate cake
 
 <en:Chocolate cakes
 fr:Moelleux au chocolat
+en:Chocolate cake w melting centre refrigerated
```

Comment:
```suggestion
en:Chocolate cake with melting centre
```

Reply:
The suggested change has not been applied.

Reply:
fixed